packages/oc-s3-riak-storage-adapter/README.md

```diff
@@ -0,0 +1,131 @@
+# RiakCS as registry storage
+
```

Comment:
Until this is stable and has proper test coverage we should add a note to mention to not use this in production yet, something like:
`⚠️ THIS ADAPTER IS IN ACTIVE DEVELOPMENT, DON'T USE IN PRODUCTION`